dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst

```diff
@@ -177,30 +182,60 @@ This tests the operation of the fit browser.
 
 1. Check that when no data are plotted the `Fit` button on the toolbar does nothing.
 
-2. Plot more than one focused run with xunit of TOF and click the `Fit` button. A simplified version of the standard mantid fit browser should now be visible.
+2. Check the Browse Filter combobox is set to ``TOF`` and click Browse. In the ``Focus`` folder of the save directory, there should be output focussed TOF files.
+   Select multiple focussed files and click Open. Back on the main interface, check the box ``Add to Plot`` and click ``Load``.
+
+3. Click the `Fit` button in the plot toolbar. A simplified version of the standard mantid fit property browser should now be visible.
 
-3. You should be able to select runs in the Settings > Workspace combo box. If you remove a run the combobox should update. Try adding some peaks (for testing purposes add different types) and a background by right-clicking on the plot. If BackToBackExponential peaks are used then the A,B parameters should be fixed automatically for ENGIN-X data.
+4. In the fit property browser, all the plotted spectra should be available in the Settings > Workspace combo box.
+   In the central ``Run Selection`` table, remove one spectrum from the plot by unticking the ``Plot`` checkbox for one row.
+   The Settings > Workspace combo box should now update and not include the removed spectrum.
 
-4. Perform a fit by clicking Fit>Fit in the fit browser. On completion of the fit, a group workspace with suffix `_fits` should have appeared in the ADS. In this group of workspaces there should be a matrix workspace for each parameter fitted (named by convention FunctionName_ParameterName), to view this right-click on the ADS and `Show Data`. For any runs not fit there should be a NaN value in the Y and E fields. In addition there is a workspace that has converted any peak centres from TOF in d-spacing (suffix `_d`). There should be an additional table called `model` that summarises the chisq value and the function string including the best-fit parameters.
+5. Right-click on the plot image and select Add Peak and add a peak to the plot. Change the peak type to BackToBackExponential and add another peak. Also add a Linear background.
```

Comment:
I thought that the default peak type in the UI was automatically set to `BackToBackExponential` (regardless of what the setting is in the the config)

Reply:
My default is for fitting on EngDiff is Lorentzian but I've maybe changed my settings at some stage. I've changed the comment slightly. I would like to ensure that the tester uses a BacktoBackExponential, but hopefully that is already added as their first peak.